src/Xamarin.Android.Tools.AndroidSdk/Sdks/AndroidSdkBase.cs

```diff
@@ -160,9 +164,13 @@ static string GetExecutablePath (string dir, string exe)
 			if (string.IsNullOrEmpty (dir))
 				return exe;
 
-			foreach (var e in ProcessUtils.ExecutableFiles (exe))
-				if (File.Exists (Path.Combine (dir, e)))
-					return e;
+			foreach (var e in ProcessUtils.ExecutableFiles (exe)) {
+				var toolPath = Path.Combine (dir, e);
+
+				if (File.Exists (toolPath))
+					return toolPath;
```

Comment:
I'm worried that you're breaking the expected return value of this function, as it will now return a full path rather than just the tool name with the proper extension appended?

Perhaps you can do a `Path.Combine(sdk.AndroidToolsBinPath, sdk.ApkAnalyzer)` further up in md-addins or wherever you need to actually use this tool, instead of relying on `sdk.ApkAnalyzer` to be a fully qualified path, if this makes sense?